import-export-cli/integration/apim/apim.go

```diff
@@ -2039,6 +2050,234 @@ func (instance *Client) registerClient(username string, password string) dcrResp
 	return jsonResp
 }
 
+// AddThrottlePolicy : Add new Throttle Policy of different policy types to APIM
+func (instance *Client) AddThrottlePolicy(t *testing.T, policy interface{}, username, password, policyType string, doClean bool) map[string]interface{} {
+	var throttlePolicyResponse map[string]interface{}
+
+	throttlePolicyURL := instance.adminRestURL + "/throttling/policies/" + policyType
+
+	data, err := json.Marshal(policy)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	request := base.CreatePost(throttlePolicyURL, bytes.NewBuffer(data))
+
```

Comment:
Let's remove these unnecessary empty lines. No use of having them. Fix in other places as well

Reply:
fixed with 0fd0d5442f47bfc4418415b38bf56c03bfc1e655